src/modules/removeTask.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-cycle
 import Store from './store.js';
 
 const removeTask = (id) => {
```

Comment:
- Good use of the `filter()` in this function to use it on the `clearAllComplete` function. 👍🏽